src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/entity/MergedCalendar.java

```diff
@@ -63,7 +80,16 @@ public void setOutputCalendarId(String outputCalendarId) {
         return calendarId;
     }
 
-    public void setCalendarId(List<CalendarId> calendarId) {
+    public void setCalendarId(
+        List<CalendarId> calendarId) {
         this.calendarId = calendarId;
     }
+
+//    public UserModel getUser() {
+//        return user;
+//    }
+//
+//    public void setUser(UserModel user) {
+//        this.user = user;
+//    }
```

Comment:
No please, no, you can delete if you don't need, if by any chance we will need t later, we can get it back from older commits.